x-pack/plugin/sql/sql-client/src/test/java/org/elasticsearch/xpack/sql/client/VersionTests.java

```diff
@@ -31,27 +31,74 @@ public void testInvalidVersion() {
         assertEquals("Invalid version format [7.1]", err.getMessage());
     }
 
-    public void testVersionFromJarInJar() throws IOException {
+
+    private static final String JAR_PATH_SEPARATOR = "!/";
+
+    private static String versionString(byte[] parts) {
+        StringBuffer version = new StringBuffer();
+        for (byte part : parts) {
+            if (version.length() > 0) {
+                version.append(".");
+            }
+            version.append(part);
+        }
+        return version.toString();
+    }
+
+    private static Path createDriverJar(byte[] parts) throws IOException {
         final String JDBC_JAR_NAME = "es-sql-jdbc.jar";
-        final String JAR_PATH_SEPARATOR = "!/";
 
         Path dir = createTempDir();
-        Path jarPath = dir.resolve("uberjar.jar");          // simulated uberjar containing the jdbc driver
-        Path innerJarPath = dir.resolve(JDBC_JAR_NAME); // simulated ES JDBC driver file
+        Path jarPath = dir.resolve(JDBC_JAR_NAME); // simulated ES JDBC driver file
 
         Manifest jdbcJarManifest = new Manifest();
         Attributes attributes = jdbcJarManifest.getMainAttributes();
         attributes.put(Attributes.Name.MANIFEST_VERSION, "1.0.0");
         attributes.put(new Attributes.Name("Change"), "abc");
-        attributes.put(new Attributes.Name("X-Compile-Elasticsearch-Version"), "1.2.3");
+        attributes.put(new Attributes.Name("X-Compile-Elasticsearch-Version"), versionString(parts));
 
         // create the jdbc driver file
-        try (JarOutputStream jdbc = new JarOutputStream(Files.newOutputStream(innerJarPath, StandardOpenOption.CREATE), jdbcJarManifest)) {}
+        try (JarOutputStream __ = new JarOutputStream(Files.newOutputStream(jarPath, StandardOpenOption.CREATE), jdbcJarManifest)) {}
```

Comment:
:-)

Reply:
Yeh, `_` is now a keyword (besides a common-ish practice to name unused vars :-) ). I just wanted to have the line within the line len limit and no extra line added.